tests/compose/runtest.sh

```diff
@@ -44,6 +46,8 @@ git clone file://${fixtures}/config
 ostree init --repo repo --mode=bare-user
 
 if "${topsrcdir}/tests/compose/test-${testname}.sh"; then
+  mkdir -p ${successdir} >&3
+  mv ${outputdir} ${successdir}/${testname} >&3
```

Comment:
Any reason why these use `>&3` ? That descriptor is specifically for writing text to the terminal/top-level invocation of `compose.sh` directly.